esy.json

```diff
@@ -14,12 +14,7 @@
   ],
   "esy": {
     "build": [
-      [ "refmterr", "jbuilder", "build", "./src/layout-test-fixed-encoding/LayoutTestFixedEncoding.exe"],
```

Comment:
These should probably go in a separate `.sh` file to perform the required builds for the sake of benchmarking. Otherwise when people run `./bench` it will fail.

Reply:
It will be generated automatically along with the library. (but not .bs.exe and js.exe)